src/wizard/ruby/index.md

```diff
@@ -7,41 +7,32 @@ type: language
 
 ## Installation {#install}
 
-Raven Ruby comes as a gem and is straightforward to install. If you are using Bundler just add this to your `Gemfile`:
+Sentry Ruby comes as a gem and is straightforward to install. If you are using Bundler just add this to your `Gemfile`:
 
 ```ruby
-gem "sentry-raven"
+gem "sentry-ruby"
 ```
 
 ## Configuration {#configure}
 
-To use Raven Ruby all you need is your DSN. Like most Sentry libraries it will honor the `SENTRY_DSN` environment variable. You can find it on the project settings page under API Keys. You can either export it as environment variable or manually configure it with `Raven.configure`:
+To use Sentry Ruby all you need is your DSN. Like most Sentry libraries it will honor the `SENTRY_DSN` environment variable. You can find it on the project settings page under API Keys. You can either export it as environment variable or manually configure it with `Sentry.init`:
 
 ```ruby
-Raven.configure do |config|
+Sentry.init do |config|
   config.dsn = '___PUBLIC_DSN___'
```

Comment:
We need to also install performance monitoring, especially in this context, as this is a brand new use of our Ruby SDK, so users shouldn't need to install tracing separately.